app/views/rubric_criteria/_rubric_criterion_level.html.erb

```diff
@@ -1,12 +1,13 @@
 <%#
 # Erb template for display rubric criterion levels in the assignment rubric manager
 %>
-<% i = 0 %>
 <%= criterion_form.fields_for :levels do |level_form| %>
   <div class="rubric_level_header">
-    <%= I18n.t("rubric_criteria.level.name")%>
-    <%= i %>
-    <% i += 1 %>
+    <%= I18n.t("rubric_criteria.level.name") + " " + level_form.index.to_s %>
+  </div>
+  <div class="destroy_checkbox">
+    <%= level_form.check_box :_destroy%>
+    <%= level_form.label :_destroy, "Delete Level " + level_form.index.to_s %>
```

Comment:
@ashih2018 everything looks good, except here you should internationalize this string.